react-components/src/components/CacheProvider/NodeCacheProvider.tsx

```diff
@@ -34,7 +34,7 @@ export const useFdmNodeCache = (): FdmNodeCacheContent => {
 
 export const useMappedEdgesForRevisions = (
   modelRevisionIds: Array<{ modelId: number; revisionId: number }>,
-  fetchViews = false,
```

Comment:
Are we sure we want this? I thought views were fetched upon clicking the nodes, and isn't necessary to fetch for all nodes e.g. in the search?